<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="pl.failure.github.GitHubSpec" tests="3" skipped="0" failures="1" errors="0" timestamp="2020-12-12T20:16:46" hostname="polpc06065" time="0.38">
  <properties/>
  <testcase name="should return" classname="pl.failure.github.GitHubSpec" time="0.364"/>
  <testcase name="should try to apply" classname="pl.failure.github.GitHubSpec" time="0.006"/>
  <testcase name="should try to clear" classname="pl.failure.github.GitHubSpec" time="0.01">
    <failure message="Too many invocations for:&#10;&#10;0 * client.put(&quot;repo-name&quot;, []) &gt;&gt; []   (1 invocation)&#10;&#10;Matching invocations (ordered by last occurrence):&#10;&#10;1 * client.put('repo-name', [])   &lt;-- this triggered the error&#10;&#10;" type="org.spockframework.mock.TooManyInvocationsError">Too many invocations for:

0 * client.put(&quot;repo-name&quot;, []) &gt;&gt; []   (1 invocation)

Matching invocations (ordered by last occurrence):

1 * client.put('repo-name', [])   &lt;-- this triggered the error


	at org.spockframework.mock.runtime.MockInteraction.accept(MockInteraction.java:70)
	at org.spockframework.mock.runtime.MockInteractionDecorator.accept(MockInteractionDecorator.java:50)
	at org.spockframework.mock.runtime.InteractionScope$1.accept(InteractionScope.java:51)
	at org.spockframework.mock.runtime.MockController.handle(MockController.java:40)
	at org.spockframework.mock.runtime.JavaMockInterceptor.intercept(JavaMockInterceptor.java:72)
	at org.spockframework.mock.runtime.CglibMockInterceptorAdapter.intercept(CglibMockInterceptorAdapter.java:30)
	at pl.failure.github.GitHub.applyUpdate(GitHub.java:99)
	at pl.failure.github.GitHubSpec.should try to clear permissions before update if requested(GitHubSpec.groovy:127)
    </failure>
  </testcase>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
